src/xmpp/session.c

```diff
@@ -320,8 +320,8 @@ session_login_success(gboolean secured)
     blocking_request();
 
     // items discovery
+    connection_request_features();
```

Comment:
Not sure this is still required. The features are requested for main domain when we handle the disco items.